packages/plugins/typescript/operations/src/visitor.ts

```diff
@@ -24,6 +24,7 @@ export interface TypeScriptDocumentsParsedConfig extends ParsedDocumentsConfig {
   immutableTypes: boolean;
   noExport: boolean;
   maybeValue: string;
+  autoSelectId: boolean;
```

Comment:
Should this also allow providing a list of potential id field names and ID scalars?

Reply:
Are there any frameworks other than Relay that rely on this behavior? I'd be happy to implement if we can come up with some reasonable use cases.